ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/monitor_telemetry_postcard_policy.schema.yml

```diff
@@ -0,0 +1,129 @@
+                valid_values: [ 1, 2 ]
+          sample:
+            type: dict
+            description: Sampling parameters.
+            keys:
+              rate:
+                type: int
+                description: Sampling rate. `rate` is preferred when both `rate` and `tcp_udp_checksum` are defined.
+                convert_types:
+                - str
+                valid_values: [ 16384, 32768, 65536 ]
+              tcp_udp_checksum:
+                type: dict
+                description: TCP/UDP parameters.
+                keys:
+                  value:
+                    type: int
+                    description: TCP/UDP checksum or IP ID value.
+                    min: 0
+                    max: 65535
+                    convert_types:
+                    - str
+                  mask:
+                    type: str
+                    description: 16 bit hexadecimal mask for TCP/UDP or IP ID with atmost 2 unset bits.
+      marker_vxlan:
+        type: bool
+        description: Enable vxlan marking using default bit 0.
+      vxlan_marking_bit:
+        type: int
+        min: 1
+        max: 31
+        convert_types:
+        - str
+      profiles:
+        type: list
+        description: Postcard telemetry profiles.
+        primary_key: name
+        items:
+          type: dict
+          keys:
+            name:
+              type: str
+              description: Profile name.
+            ingress_sample_policy:
+              type: str
+      sample_policies:
+        type: list
+        primary_key: name
+        items:
+          type: dict
+          keys:
+            name:
+              type: str
+            match_rules:
+              type: list
+              primary_key: name
+              items:
+                type: dict
+                keys:
+                  name:
+                    type: str
+                  type:
+                    type: str
+                    description: IP address version.
+                    required: true
+                    valid_values: [ ipv4, ipv6 ]
+                  destination_prefix:
+                    type: str
+                    description: |-
+                      IPv4 Network/Mask or IPv6 Network/Mask. Host part of prefix must be zero.
+                      eg. 10.3.3.0/24
+                  source_prefix:
+                    type: str
+                    description: |-
+                      IPv4 Network/Mask or IPv6 Network/Mask. Host part of prefix must be zero.
+                      eg. 10.3.3.0/24
+                  protocols:
+                    type: list
+                    primary_key: protocol
+                    items:
+                      type: dict
+                      keys:
+                        protocol:
+                          type: str
+                          valid_values: [ tcp, udp ]
+                        destination_ports:
+                          type: list
```

Comment:
You are missing an `items` key for the list, describing the individual list items. Same for source_ports.

Reply:
Fixed.